newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -24,6 +24,34 @@ export const splittedProjectFolderNames = [
   'eventsFunctionsExtensions',
 ];
 
+const deleteExistingFilesFromDirs = (
+  project: gdProject,
+  projectPath: string
+) => {
+  const entries = fs.readdirSync(projectPath);
+
+  if (project.isFolderProject()) {
+    //If multiFile enabled in settings and directories exist!
+    entries.forEach(entry => {
+      let dirPath = projectPath.concat('\\', entry);
+      if (
+        fs.statSync(dirPath).isDirectory() &&
+        splittedProjectFolderNames.includes(entry)
+      ) {
+        const filenames = fs.readdirSync(dirPath);
+        filenames.forEach(file => {
+          let result = dirPath.concat('\\', file);
```

Comment:
You could rename `result` with a name that actually represents what it is: `filePath` or `fileToRemovePath`